### PR TITLE
👜 next+header in UI 👜

### PR DESCRIPTION
**Pull request by uzproblue**
New react + next app initialized
Header UI from Figma created

### CODE REVIEW

**Comment by jbampton** [comment]
You need to install `pre-commit` so that you can run some tests locally.

The `pre-commit` test is failing here.

https://github.com/One-Language/One/blob/master/CONTRIBUTING.md

**Review by jbampton — CHANGES_REQUESTED** [review]
MD041 Error - First line in file should be a top level header

https://github.com/One-Language/Website/pull/34/checks?check_run_id=2741855499

https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md041---first-line-in-file-should-be-a-top-level-header

https://github.com/One-Language/Website/blob/master/.pre-commit-config.yaml

https://pre-commit.com/

File: website/pages/api/hello.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
+
+export default (req, res) => {
+  res.status(200).json({ name: 'John Doe' })
```
Comment (jbampton) [inline website/pages/api/hello.js:4]:
```suggestion
  res.status(200).json({ name: 'John Doe' });
```

Comment (jbampton) [inline website/pages/api/hello.js]:
```suggestion
  res.status(200).json({ name: 'John Doe' });
```

**Comment by BaseMax** [comment]
John, it's ES6 code. so need to create `.jshintrc` file with:
```
{
  "esversion": 6
}
```
And I guess errors will solve!

**Review by devkadirselcuk — APPROVED** [review]

**Pull request merged**